src/magellan_motion/src/path_planner/planner.cpp

```diff
@@ -199,7 +205,7 @@ Path PathPlanner::plan(Point goal) {
     return p;
 }
 
-bool PathPlanner::isFree(double x, double y) {
+bool PathPlanner::isFree(int x, int y) {
     int mapWidth = _map.info.width;
     double mapResolution = std::roundf(_map.info.resolution * 100.0) / 100.0;
 
```

Comment:
So in this function, we can do this two ways. First, we convert the x, y pose back into a floating-point number then back into an integer using the same logic I used before. The second option is to use the integer numbers directly and use them directly on line 238, but using just integers. This assumes the floating precision we use here is the same as the resolution the occupancy grid is in, which is a safe assumption to make.

Reply:
Added temps for floating point precision in latest commits